pkg/uefi/meregion_test.go

```diff
@@ -91,3 +92,25 @@ func TestFindFPTSignature(t *testing.T) {
 		})
 	}
 }
+
+func TestParseFPT(t *testing.T) {
+	t.Run("parse ME FPT", func(t *testing.T) {
+		meRegion, err := os.ReadFile("../../data/PurleySiliconBinPkg/MeFirmware/IgnitionFirmware/MeRegion.bin")
+		if err != nil {
+			t.Errorf("%q", err)
+		}
+		expectedEntries := 10
+
+		fpt, err := NewMEFPT(meRegion)
+		if err != nil {
+			t.Errorf("expected no error reading a valid ME FPT")
```

Comment:
might as well make this 
t.Fatalf("NewMEFPT: got %v, want nil", err)

Reply:
IIUC correctly, it may be against conventions, though technically, we may get both and error _and_ a result, right? I'd want to keep the test running to the end, in that case, so that it may uncover another possible error.